src/index.html

```diff
@@ -1,15 +1,170 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <!--<h1>HTML Form</h1>-->
+    <form
+    id="btn_save"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <section class="section">
+        <fieldset>
+          <legend>Personal information</legend>
+          <div class="block">
+            <label for="surname">Surname:</label>
+            <input
+            id="surname"
+            type="text"
+            name="inp_surname"
+            >
```

Comment:
```suggestion
            <input
              id="surname"
              type="text"
              name="inp_surname"
            >
```

Let's format all your code below correctly